django/views/templates/default_urlconf.html

```diff
@@ -104,21 +92,23 @@
               animation: none;
             }
           }
-          h2 {
-            font-size: 22px;
-            max-width: 500px;
+          h1 {
+            font-size: 1.375rem;
+            max-width: 32rem;
             margin: 5px auto 0;
           }
           main p {
-            font-size: 16px;
-            line-height: 20px;
-            max-width: 410px;
+            line-height: 1.25;
+            max-width: 26rem;
             margin: 15px auto 0;
-            color: #888888;
           }
           footer {
+            display: grid;
+            grid-template-columns: 1fr 1fr 1fr;
+            gap: 5px;
```

Comment:
The layout shifted slightly with this, because the previous implementation also had padding to the right of the last item in the row.